google/cloud/security/inventory/inventory_loader.py

```diff
@@ -18,7 +18,7 @@
 
 Usage:
   $ forseti_inventory \\
-      --inventory_groups <true|false> (optional) \\
+      --pipeline_config (required) \\
       --groups_service_account_key_file <path to file> (optional)\\
```

Comment:
Then, let's do this in another PR.  The proposed change should happen in conjunction with the deployment script, and in isolation of this PR.